From ce2d7c0cbe699ef829f20ef40845c921fd3fc249 Mon Sep 17 00:00:00 2001
From: Kalyan Alle <kalyan.alle@amd.com>
Date: Wed, 25 Apr 2018 12:48:35 +0530
Subject: [PATCH 5703/5855] compilation issues resolved v8

This patch resolves the compilation issues prompted during
porting from branch: amd-18.10, repo: brahma/ec/linux
till commit : 4f3ad6a4e70c47bed615207be0aabb27ab2d5180

Signed-off-by: kalyan alle <kalyan.alle@amd.com>
---
 drivers/gpu/drm/amd/amdkfd/kfd_priv.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/gpu/drm/amd/amdkfd/kfd_priv.h b/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
index 59d01f4..870fb55 100644
--- a/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
+++ b/drivers/gpu/drm/amd/amdkfd/kfd_priv.h
@@ -24,6 +24,7 @@
 #define KFD_PRIV_H_INCLUDED
 
 #include <linux/hashtable.h>
+#include <linux/version.h>
 #include <linux/mmu_notifier.h>
 #include <linux/mutex.h>
 #include <linux/types.h>
-- 
2.7.4

